vendor/github.com/elastic/go-libaudit/CHANGELOG.md

```diff
@@ -0,0 +1,64 @@
+# Change Log
```

Comment:
I assume you could put these under `ignore`: https://github.com/elastic/beats/blob/master/metricbeat/module/docker/vendor/vendor.json#L3

@vjsamuel I remember this is something we discussed before and will have that problem more and more often. How do we deal with dependencies that are already available on the top level vendor dir?